include/kernel.h

```diff
@@ -5217,6 +5217,13 @@ static inline void k_cpu_idle(void)
  * Similar to k_cpu_idle(), but called with interrupts locked if operations
```

Comment:
Could you re-word this sentence, as well @pabigot , so it reads:
`Similar to k_cpu_idle(), but it must be called with interrupts locked` ? Honestly, I don't see the reason for the "if operations must be..."

Reply:
OK, since the only way you can get a `key` to pass is by `irq_lock()` that rewording makes sense.  Done.